provision-contest/ansible/emergency.yml

```diff
@@ -0,0 +1,60 @@
+---
+- hosts: domserver
+  tasks:
+    - name: Disable webserver to stop submissions
+      tags: powerloss
+      service:
+        name: nginx
+        state: stopped
+
+- hosts: all
+  tasks:
+    - name: Disable GDM autologin
```

Comment:
do we also have the reverse somewhere?

Reply:
I would say the reverse is running {domjudge,judgehost,admin}.yml again as we want to make sure we're in a good state.

I thought about reversing the logic but we have enough dependencies so I think doing a "full" setup run again (which will still be fast) is the safer option here.

Reply:
Reverse is now added to the other playbooks